portfolio/pom.xml

```diff
@@ -23,11 +23,13 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+
```

Comment:
nit: i don't think there should be spaces between the dependencies

Reply:
I think its better for readability. I will change it if you have strong preference.